lib/commands/recordscreen.js

```diff
@@ -0,0 +1,99 @@
+import { sleep } from 'asyncbox';
+
+let commands = {}, extensions = {};
+
+const RETRY_PAUSE = 1000;
+
+/**
+* record the display of devices running Android 4.4 (API level 19) and higher.
+* It records screen activity to an MPEG-4 file. Audio is not recorded with the video file.
+*
+* @param filePath the video file name
+*                 for example, "/sdcard/demo.mp4"
+* @param videoSize the format is widthxheight.
+*                  if it is "default", the default value is the device's native display resolution (if supported),
+*                  1280x720 if not. For best results,
+*                  use a size supported by your device's Advanced Video Coding (AVC) encoder.
+*                  for example, "1280x720"
+* @param timeLimit the maximum recording time, in seconds. if it is -1, the default and maximum value is 180 (3 minutes).
+* @param bitRate the video bit rate for the video, in megabits per second.
+*                if it is -1, the default value is 4Mbps. You can increase the bit rate to improve video quality,
+*                but doing so results in larger movie files.
+*                for example, 6000000
+*
+*/
+commands.startRecordingScreen = async function ( filePath, videoSize, timeLimit, bitRate ) {
+  let cmd, data, fileExists, loop;
+
+  if (this.isEmulator()) {
+    throw new Error('Android screen recording does not work on emulators');
+  }
+  
+  data = await this.adb.getApiLevel();
+
+  //this function is suppported on the device running android 4.4(api level 19)
+  if (data < 19) throw new Error('The screenrecord command is able to record the display of devices running Android 4.4 (API level 19) and higher.');
+
```

Comment:
I searched the pattern in appium. I found that 
in case of the try{}catch() pattern, it usually used log.errorAndThrow.
so, I did not use log.errorAndThrow in this code, but I applied it in the try{}catch patten in my code. thank you!